### PR TITLE
changes made for czentrix

### PR DESCRIPTION
**Pull request by Vidyaaa24**

### CODE REVIEW

File: src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public class T_104CoMoOutboundCallDetails {
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long iD;
 	private Integer uSRMappingID;
-	private String czentrixCallID;
+	private String callCentreID;
```
Comment (harimohanr) [inline src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java:42]: @Vidyaaa24 Please change this property to callId. It is not callCentreId. if you look at the original property, it was czentrixCallId.

File: src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java [hunk 2]
```diff
@@ -59,12 +59,12 @@ public T_104CoMoOutboundCallDetails() {
 		// TODO Auto-generated constructor stub
 	}
 
-	public T_104CoMoOutboundCallDetails(Long iD, Integer uSRMappingID, String czentrixCallID, String callType,
+	public T_104CoMoOutboundCallDetails(Long iD, Integer uSRMappingID, String callCentreID, String callType,
```
Comment (harimohanr) [inline src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java:62]: Please change this property to callId. It is not callCentreId. if you look at the original property, it was czentrixCallId.

**Review by harimohanr — CHANGES_REQUESTED** [review]
@Vidyaaa24 Please change the property to callId. It is not callCentreId. if you look at the original property, it was czentrixCallId.

**Review by Anjana2310 — CHANGES_REQUESTED** [review]
@Vidyaaa24 please modify callCentreID to callId in all its references

File: src/main/java/com/iemr/helpline104/service/bloodRequest/BloodRequestServiceImpl.java [hunk 3]
```diff
@@ -176,7 +176,7 @@ private T_BloodRequest update(T_BloodRequest t_bloodRequest) throws Exception
 	 * Exception {
 	 * 
 	 * return bloodOutboundDetailsRepository.update(t_bloodOutboundDetails.
-	 * getBloodOutboundDetailID(),t_bloodOutboundDetails.getCzentrixCallID()); }
+	 * getBloodOutboundDetailID(),t_bloodOutboundDetails.getCallCentreID()); }
```
Comment (Anjana2310) [inline src/main/java/com/iemr/helpline104/service/bloodRequest/BloodRequestServiceImpl.java:179]: @Vidyaaa24 Please remove the whole commented out code

File: src/main/java/com/iemr/helpline104/repository/comoOutbound/CoMoOutboundCallRepository.java [hunk 4]
```diff
@@ -38,6 +38,6 @@ public interface CoMoOutboundCallRepository extends CrudRepository<T_104CoMoOutb
 
 	@Transactional
 	@Modifying
-	@Query("update T_104CoMoOutboundCallDetails set czentrixCallID = :czentrixCallID WHERE iD = :iD")
-	public Integer update(@Param("iD") Long iD, @Param("czentrixCallID") String czentrixCallID);
+	@Query("update T_104CoMoOutboundCallDetails set callCentreID = :callCentreID WHERE iD = :iD")
```
Comment (Anjana2310) [inline src/main/java/com/iemr/helpline104/repository/comoOutbound/CoMoOutboundCallRepository.java:41]: @Vidyaaa24 Please modify as discussed

**Review by Anjana2310 — CHANGES_REQUESTED** [review]
@Vidyaaa24 Changes to be done:1.Change to callId in CoMoOutboundCallrepository.java 2.Remove commented out code instead of modifying in BloodRequestServiceImpl.java

**Review by harimohanr — APPROVED** [review]
@Vidyaaa24 Looks okay to me

**Review by Anjana2310 — APPROVED** [review]
@Vidyaaa24 Looks good to me

**Pull request merged**